saleor/dashboard/staff/views.py

```diff
@@ -48,14 +49,22 @@ def staff_details(request, pk):
 @staff_member_required
 @permission_required('userprofile.edit_staff')
 def staff_create(request):
-    staff = User()
+    try:
+        staff = User.objects.get(email=request.POST.get('email'))
```

Comment:
Can we use `get_or_create` here?

Reply:
This action is made before checking if ```form.is_valid()```, so we shouldn't create a user here. We can move that to form's saving method, but then it would require changing form instance.

Reply:
True, so let's keep it.